tardis/adapters/sites/kubernetes.py

```diff
@@ -0,0 +1,176 @@
+import kubernetes_asyncio.client
+
+from ...configuration.configuration import Configuration
+from ...exceptions.tardisexceptions import TardisError
+from ...interfaces.siteadapter import SiteAdapter
+from ...interfaces.siteadapter import ResourceStatus
+from ...utilities.attributedict import AttributeDict
+from ...utilities.staticmapping import StaticMapping
+
+from kubernetes_asyncio import client
+
+from functools import partial
+from datetime import datetime
+from contextlib import contextmanager
+
+
+class KubernetesAdapter(SiteAdapter):
+    def __init__(self, machine_type: str, site_name: str):
+        self._configuration = getattr(Configuration(), site_name)
+        self._machine_type = machine_type
+        self._site_name = site_name
+
+        self.aConfiguration = kubernetes_asyncio.client.Configuration(
```

Comment:
`self.aConfiguration` is only used in the property method `client` in case the current client is `None`. Please consider moving the creation of `aConfiguration` there. Or do you foresee any reason that this configuration might be changed from somewhere else?
Further, you might consider using snakecase for naming your variables as this better matches current naming conventions.

Reply:
Please note that you imported `client` and `kubernetes_asyncio.client` and you are currently using both forms. As @maxfischer2781 suggested, please make it more explicit, e.g. with `k8s_client` and remove the import `import kubernetes_asyncio.client`.

Reply:
Moved the creation of a_configuration, used snakecase for naming all variables and changed import